open_academy/__manifest__.py

```diff
@@ -8,7 +8,8 @@
     "license": "LGPL-3",
     "version": "15.0.1.0.1",
     "depends": [
-        "base"
+        "base",
```

Comment:
Remove this, board app depends on base module. Remember avoid recursive dependencies.

Reply:
Hi @edy1192, if I remove the _board_ dependency, I get this error message: **Model not found: board.board**. 

Should I do something else to make it work?

Reply:
I meant base module, not board.